examples/table/table.out

```diff
@@ -10,7 +10,7 @@ Employee 1: id=1 name=John salary=300.5
 Information of the removed Employee: id=2 name=Bella salary=500.5
 Employee table keys: 1 3 4
 Employee table to list: id=1 name=John salary=300.5 id=3 name=Peter salary=750.0 id=4 name=Max salary=900.0
-Employees with salary < 400.0: John
+Employees with salary < 400.0: John 
```

Comment:
Due to this : 
https://github.com/ballerina-platform/ballerina-distribution/pull/608/files#diff-59268ffca29146e6b7b30a20bcf9af5cL70